docs/assets/css/site.css

```diff
@@ -443,6 +443,7 @@ code {
 */
 .flip-container .front,
 .flip-container .back {
+    height: 240px;
```

Comment:
If you look in bower.json and update the `caf-templates` to point to the latest release:

```json
"caf-templates": "https://svsartifactory.swinfra.net:443/artifactory/api/npm/company-central-npm/caf-templates-2.1.0.tgz"
```

You should then be able to remove this style change as it will be available in the caf-templates package.

Let me know if you have any issues, thanks.